torchgeo/models/croma.py

```diff
@@ -0,0 +1,649 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Code based on https://github.com/antofuller/CROMA under MIT License
```

Comment:
Need to check but I don't think we can just copy-n-paste without also copying the entire license (including who owns the copyright). I wish we had a legal team to answer questions like this...